packages/components/src/utils/config-values.js

```diff
@@ -27,6 +30,12 @@ const CONTROL_PROPS = {
 	controlHeightXLarge: `calc( ${ CONTROL_HEIGHT } * 1.4 )`,
 };
 
+const SLIDER_PROPS = {
```

Comment:
I wonder if it makes sense to add these values (and the ones above) in a shared config, if they're going to be used only in one component.

@mirka, WDYT ?

Reply:
These were a pretty poor effort at bringing in the UI values from G2 without knowing where best they belong in Gutenberg. As part of the general color/styles clean up in https://github.com/WordPress/gutenberg/commit/9acaa77cdb788d2993824b353dfc41fea366e6c9, I've removed these.